src/main/java/com/example/demo/matcher/models/Order.java

```diff
@@ -18,6 +18,11 @@ public class Order {
     private final OrderAction action;
     private final Long datetime;
 
+    public static final int minPrice = 0;
+    public static final int maxPrice = 1000000000;
```

Comment:
Where is this constant used?

Reply:
setting @min and @max validation constraints in the NewOrderParams class, which is responsible for receiving information about the new order at the endpoint

Reply:
![image](https://user-images.githubusercontent.com/118171430/211869425-78d7cb0f-4d68-425e-b6d3-8d2756b74e56.png)